tests/python_client/testcases/test_index.py

```diff
@@ -1547,6 +1548,261 @@ def test_index_with_string_field_empty(self):
 
         collection_w.create_index(default_string_field_name, default_string_index_params, index_name=index_name2)
         assert collection_w.has_index(index_name=index_name2)[0] == True
-
         collection_w.drop_index(index_name=index_name2)
-        assert len(collection_w.indexes) == 0
\ No newline at end of file
+        assert len(collection_w.indexes) == 0
+    
+class TestIndexDiskann(TestcaseBase):
+    """
+    ******************************************************************
+      The following cases are used to test create index about diskann
+    ******************************************************************
+    """
+    @pytest.fixture(scope="function", params=[False, True])
+    def _async(self, request):
+        yield request.param
+
+    def call_back(self):
+        assert True
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_create_index_with_diskann_normal(self):
+        """
+        target: test create index with diskann
+        method: 1.create collection and insert data 
+                2.create diskann index , then load data
+                3.search successfully
+        expected: create index successfully
+        """
+        c_name = cf.gen_unique_str(prefix)
+        collection_w = self.init_collection_wrap(name=c_name)
+        data = cf.gen_default_list_data()
+        collection_w.insert(data=data)
+        index, _ = self.index_wrap.init_index(collection_w.collection, default_float_vec_field_name, ct.default_diskann_index)
+        log.info(self.index_wrap.params)
+        cf.assert_equal_index(index, collection_w.indexes[0])
+        collection_w.load()
+        assert collection_w.num_entities == default_nb
+        vectors = [[random.random() for _ in range(default_dim)] for _ in range(default_nq)]
+        search_res, _=collection_w.search(vectors[:default_nq], default_search_field,
```

Comment:
Here should format this line.

And why not use search result check function directly?

Reply:
Already modified